vm/src/vm.rs

```diff
@@ -836,6 +830,26 @@ impl VirtualMachine {
         }
     }
 
+    fn jump_if(
+        &mut self,
+        label: &bytecode::Label,
+        obj: PyObjectRef,
+        jump_on: bool,
```

Comment:
Hmm, I'm not a big fan of functions with booleans as arguments.

Reply:
Me neither. jump_on seems okay (it's used as a value, not as a flag), but pop_on bothers me. The error handling means that this helper function doesn't help much unless the pop_on moves into it too though.

I'll simply kill this function and return to repetitive definitions of JumpIf... if you prefer?

Or other suggestions?

Reply:
I think having some repetition is okay if it makes the code better to understand. I have no other suggestions here :smile: